docker/docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
     networks:
       - demo
     healthcheck:
-      test: [ "CMD-SHELL", "curl http://localhost:12800/internal/l7check" ]
+      test: ["CMD-SHELL", "curl --silent --fail http://localhost:12800/internal/l7check || exit 1"]
```

Comment:
why this check example - silent fail ?

Reply:
This was made when ui was not working, will edit this again !